src/chronometer.js

```diff
@@ -1,33 +1,54 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
-  start(printTimeCallback) {
-    // ... your code goes here
+  start(printTimeCallback){
+  this.intervalId = setInterval(()=>
+  this.currentTime += 1
+  ,1000)
+ /* function stringRetorned(){
+    return printTimeCallback
+  }*/ 
   }
 
   getMinutes() {
-    // ... your code goes here
+    const minuntesCurrent = Math.floor(this.currentTime / 60)
+    return minuntesCurrent
   }
 
   getSeconds() {
-    // ... your code goes here
+    const secondsCurrent = this.currentTime % 60
+    return secondsCurrent
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let arr = []
+    if(value === 0 || value === 1 ||value === 2 || value === 3 || value === 4 || value === 5 || value === 6 || value === 7 || value === 8 || value === 9){
+      arr = [0]
+      arr.push(value)
+    }else{
+      arr.push(value)
+    }
+    let twoDigitNumber = arr.join('')
+    return twoDigitNumber
   }
```

Comment:
Aqui podemos só transformar o value em String primeiro, conforme o lab pede: value.toString( );
com o if, podemos pensar numa lógica mais simples, num código mais enxuto e claro...veja só
se o value for identico a zero, retorna '00',
se o value for maior ou igual a 10, retorna o valor transformado em string;
senao, aí retornamos '0' + o valor transformado em string. 
Tenta essas alterações, somente para o código ficar mais simples e fácil de ler :)